tokens/src/commands.rs

```diff
@@ -98,8 +99,14 @@ type StakeExtras = Vec<(Keypair, Option<DateTime<Utc>>)>;
 pub enum Error {
     #[error("I/O error")]
     IoError(#[from] io::Error),
-    #[error("CSV error")]
+    #[error("CSV file seems to be empty")]
+    CsvIsEmptyError,
+    #[error("Bad input data for pubkey")]
     CsvError(#[from] csv::Error),
+    #[error("Bad input data for pubkey")]
+    BadInputPubkeyError(#[from] pubkey::ParsePubkeyError),
+    #[error("Bad input data for lockup date")]
+    BadInputLockupDate(#[from] chrono::ParseError),
```

Comment:
Perhaps instead of or as well as passing the parser error, we should update these to print the problematic pubkey/lockup string. What do you think?
That could look something like:
```
#[error("Bad input data for pubkey: {input} {err}")]
BadInputPubkeyError{
    input: String,
    err: pubkey::ParsePubkeyError,
},
```